core/matrix/batch_csr.cpp

```diff
@@ -181,6 +183,29 @@ void Csr<ValueType, IndexType>::apply_impl(const MultiVector<ValueType>* alpha,
 }
 
 
+template <typename ValueType, typename IndexType>
+void Csr<ValueType, IndexType>::scale_add(
+    ptr_param<const MultiVector<ValueType>> alpha,
+    ptr_param<const batch::matrix::Csr<ValueType, IndexType>> b)
+{
+    GKO_NOT_IMPLEMENTED;
```

Comment:
I don't think that is relevant at all here. We use `GKO_NOT_IMPLEMENTED` because some function has to be defined, maybe due to interface requirements or our kernel structure. But none of that applies here. This function simply doesn't need to exist, so there is zero use to declare and define this function. 
If we want to implement it at a later point we can still do so, nothing would stop us if we remove the function right now.